fabric-entity-attribute-api-v1/src/main/java/net/fabricmc/fabric/mixin/entity/attribute/DefaultAttributeRegistryMixin.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.mixin.entity.attribute;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.spongepowered.asm.mixin.Final;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Mutable;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+import net.minecraft.entity.EntityType;
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.entity.attribute.DefaultAttributeContainer;
+import net.minecraft.entity.attribute.DefaultAttributeRegistry;
+
+import net.fabricmc.fabric.impl.entity.attribute.FabricEntityAttributeRegistry;
+
+@Mixin(DefaultAttributeRegistry.class)
+public abstract class DefaultAttributeRegistryMixin {
+	@Shadow
+	@Final
+	@Mutable
```

Comment:
```suggestion
	@Mutable
```
`@Final` is unnecessary and has no purpose when `@Mutable` is present.

Reply:
Uh, isn't mixin javadoc saying [`Use in conjunction with {@link Final}`](https://github.com/FabricMC/Mixin/blob/5442b78e67d09139934b85019237b5a189cf005a/src/main/java/org/spongepowered/asm/mixin/Mutable.java#L35)?

Reply:
Humm, I have never used it with `@Final`, but yeah you should go with what the docs say.
Rip now I have a bunch of unspecified behaviour to fix :D